OrmTester/OrmTesterLib/Resources/Resources.resx

```diff
@@ -142,7 +142,7 @@
     <value>Missing TestResult Collection. StatisticCalculators needs test results from Entity Framework and NHibernate both.</value>
   </data>
   <data name="None" xml:space="preserve">
-    <value>None Relationship</value>
```

Comment:
To jest Resource do tego co jest zapisane w statistic parameters? jeśli tak to najlepiej by było No relationship czy coś takiego

Reply:
chciałem skrócić, ale jak będzie całe okno to to przywrócę

Reply:
Już jest całe okno zrzuć SB mastera :)